repo/index.mjs

```diff
@@ -1,56 +1,66 @@
 export default async function (plop) {
-  plop.setGenerator("repo:dotfiles", {
-    description: "General configuration through .dotfiles",
-    prompts: [],
-    actions: [
+  plop.setGenerator("repo:init", {
+    description: "Init a repo with general configuration",
+    prompts: [
       {
-        type: "addMany",
-        destination: "./",
-        base: "templates",
-        templateFiles: "**/.*",
-        globOptions: { dot: true },
-        verbose: true,
+        type: "checkbox",
+        name: "initOptions",
+        message: "What would you like to initialise?",
+        choices: [
+          { name: ".dotfiles", value: "dotfiles" },
+          { name: "GitHub templates", value: "github" },
+          { name: "Repo documentation", value: "docs" },
+        ],
       },
     ],
-  });
+    actions: function (data) {
+      const actions = [];
 
-  plop.setGenerator("repo:github", {
-    description: "General configuration through .dotfiles",
-    prompts: [],
-    actions: [
-      {
-        type: "addMany",
-        destination: ".github",
-        base: "templates/.github",
-        templateFiles: "**",
-        globOptions: { dot: true },
-        verbose: true,
-      },
-    ],
-  });
+      console.log(data);
```

Comment:
Is this console necessary or just for testing?

Reply:
Good catch!